front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/CraftComponents/OpenToParticipation/ProjectCarrousel/ScrollButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { Box, Icon, colors } from '@citizenlab/cl2-component-library';
+
+interface Props {
+  variant: 'left' | 'right';
+  onClick: () => void;
+}
+
+const ScrollButton = ({ variant, onClick }: Props) => {
+  return (
+    <Box
+      as="button"
+      className="scroll-button"
+      position="absolute"
+      left={variant === 'left' ? '8px' : undefined}
+      right={variant === 'right' ? '8px' : undefined}
+      top="120px"
+      borderRadius="30px"
+      bgColor="white"
+      w="52px"
+      h="52px"
+      zIndex="3"
+      border={`1px solid ${colors.divider}`}
+      onClick={(e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        onClick();
+      }}
```

Comment:
Does this need to be aria-hidden if it's not meant to be accessible by keyboard? :)

Reply:
Yes, good point!